packages/discord.js/src/structures/Widget.js

```diff
@@ -83,6 +83,17 @@ class Widget extends Base {
     this._patch(data);
     return this;
   }
+  /**
+   * Get Guild Widget Image
+   * @param {?WidgetStyleOptions} style Style of widget
+   * @returns {Promise<Widget>}
+   */
+  fetchImage(style) {
```

Comment:
This method isn't fetching anything. Should be called `getImageURL` or something